### PR TITLE
Highlight subtree

### PR DESCRIPTION
**Pull request by Evgenios95**
Screenshot 2022-05-10 at 13 42 24

### CODE REVIEW

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ export const StyledNode = styled.div`
   border: ${({ isHighlighted }) => {
     return isHighlighted ? '1.5px solid black' : '1px solid black';
   }};
+  box-shadow: ${({ isHighlighted }) => {
```
Comment (piwa87) [inline packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:35]: This is a cool line of code!

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js [hunk 2]
```diff
@@ -32,6 +32,9 @@ export const StyledNode = styled.div`
...
+    return isHighlighted ? '0px 0px 16px 2.5px rgba(255,194,25,0.76)' : 'none';
```
Comment (piwa87) [inline packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:36]: Are you sure about 255,194,24,0.76? 
If yes, let's do it!

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js [hunk 3]
```diff
@@ -32,6 +32,9 @@ export const StyledNode = styled.div`
...
+  }};
```
Comment (piwa87) [inline packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:37]: Nice!

**Review by piwa87 — APPROVED** [review]
Go!

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js (hunk 1 above)
Comment (Evgenios95) [reply packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:35]: Thanks, this pattern is around 113 years old

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js (hunk 2 above)
Comment (Evgenios95) [reply packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:36]: I was thinking 255,194,24,0.73

File: packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js (hunk 3 above)
Comment (Evgenios95) [reply packages/bratus-app/src/components/ComponentNode/ComponentNode.sc.js:37]: Thanks appreciate it

**Pull request merged**